Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/weatherInput/index.tsx

```diff
@@ -8,7 +13,7 @@ const WeatherInput = () => {
         type="range"
         min="0"
         max="100"
-        value="50"
```

Comment:
You should be using controlled inputs here, let's say we have add some function in App.tsx that runs setWeather(12), would the slider be updated to reflect the change?